### PR TITLE
[7.4-only]LPS-102102 Remove modal from compat-layer and fix legacy code

### PR DESCRIPTION
**Pull request by eduardoallegrini**
Update for compat-layer:modals

### CODE REVIEW

File: modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/META-INF/resources/css/ApplicationsMenu.scss [hunk 1]
```diff
@@ -2,13 +2,10 @@
 
 .applications-menu-modal {
 	.modal-full {
```
Comment (eduardoallegrini) [inline modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/META-INF/resources/css/ApplicationsMenu.scss:4]: If not used anywhere else we might want to replace `modal-full` with `modal-dialog`

Comment (victorg1991) [reply modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/META-INF/resources/css/ApplicationsMenu.scss:4]: Sure if the styles are preserved I think we can replace it :)

Comment (eduardoallegrini) [reply modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/META-INF/resources/css/ApplicationsMenu.scss:4]: Updating
image

**Comment by marcoscv-work** [comment]
LGTM, we need to rebase and send!!!

**Comment by eduardoallegrini** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by marcoscv-work** [comment]
https://github.com/liferay-frontend/liferay-portal/pull/567

**Pull request closed without merging**